src/org/omegat/gui/main/MainWindow.java

```diff
@@ -461,9 +462,13 @@ public void displayErrorRB(final Throwable ex, final String errorKey, final Obje
             }
 
             statusLabel.setText(msg);
-            String fulltext = msg;
+            StringBuilder fulltext = new StringBuilder(msg);
             if (ex != null) {
-                fulltext += "\n" + ex.toString();
+                fulltext.append("\n");
+                for (String s : ex.getLocalizedMessage().split("\\R")) {
+                    fulltext.append(WordUtils.wrap(s, 150, "\n    ", true));
```

Comment:
I wonder if instantiating JOPtionPane and overriding its getMaxCharactersPerLineCount() to provide the limit wouldn't be sturdier.
See [https://docs.oracle.com/javase/8/docs/api/javax/swing/JOptionPane.html#getMaxCharactersPerLineCount--](https://docs.oracle.com/javase/8/docs/api/javax/swing/JOptionPane.html#getMaxCharactersPerLineCount--)